### PR TITLE
Improve Tools setup file

### PR DESCRIPTION
**Pull request by cbur24**
### Proposed changes
The `setup.py` file for `deafrica_tools` was insufficient to the task. This PR improves the file by ensuring all the necessary packages are included.

I've also removed the `segmentation.py` module since it can no longer work on the sandbox, and I commented out some older, now unused code on the `load_era5.py` module

### CODE REVIEW

File: Tools/setup.py [hunk 1]
```diff
@@ -1,36 +1,95 @@
...
-['dask-ml>=1.8.0,<2.0.0',
- 'datacube>=1.8.3,<2.0.0',
- 'geopandas>=0.8.2,<=0.9.0',
- 'joblib>=1.0.1,<2.0.0',
- 'matplotlib>=3.3.4,<4.0.0',
- 'scikit-learn>=0.24.1,<0.25.0',
- 'tqdm>=4.57.0,<5.0.0']
+# Where are we?
+if 'sandbox' in os.getenv('JUPYTER_IMAGE', default=''):
+    IS_DEA=True
```
Comment (NikitaGandhi) [inline Tools/setup.py:12]: can this be `IS_DEAFRICA_SANDBOX = True`

File: Tools/setup.py [hunk 2]
```diff
@@ -1,36 +1,95 @@
...
-    'package_data': package_data,
-    'install_requires': install_requires,
-    'python_requires': '>=3.6,<4.0',
+    'description': DESCRIPTION,
+    'long_description': long_description,
+    'author': AUTHOR,
+    'author_email': EMAIL,
+    'python_requires': REQUIRES_PYTHON,
+    'url': URL,
+    'install_requires': REQUIRED if not IS_DEA else [],
```
Comment (NikitaGandhi) [inline Tools/setup.py:91]: update this line with above change

Comment (cbur24) [reply Tools/setup.py]: Done

**Comment by NikitaGandhi** [comment]
LGTM. Let's confirm with Matthew?

**Pull request merged**